libavcodec/jpeg2000dec.c

```diff
@@ -520,7 +538,22 @@ static int get_cox(Jpeg2000DecoderContext *s, Jpeg2000CodingStyle *c)
     }
 
     c->cblk_style = bytestream2_get_byteu(&s->g);
+
```

Comment:
For submission to FFmpeg, code style modifications should be collected into a separate commit.